File: App MVC/App/App/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace App.Controllers
{
    public class HomeController : Controller
    {
        // GET: Home
        public ActionResult Index()
        {
            return View();
        }

        public object SendAlert()
        {


            //insérer un objet alerte dans la base de données


            return View();
        }

    }
}

File: App MVC/WEB_APP_GESTION_HORAIRE/WEB_APP_GESTION_HORAIRE/Controllers/HomeController.vb
Imports System.Web.Mvc
Imports System.Linq

Namespace Controllers
    Public Class HomeController
        Inherits Controller

        ' GET: Home
        Function Index() As ActionResult
            Return View()

        End Function








        Function GererHoraires()
            If Session("AdminRights") Then
                Return View()
            Else Return View("~/Views/Shared/PermissionDenied.vbhtml")
            End If

        End Function

        Function VoirHoraires()
            If Session("AdminRights") Then
                Return View()
            Else Return View("~/Views/Shared/PermissionDenied.vbhtml")
            End If
        End Function



        Function MonHoraire()
            If Session("UserLogged") Then
                Return View()
            Else Return View("~/Views/Shared/PermissionDenied.vbhtml")
            End If
        End Function



    End Class
End Namespace